internal/cloudlog/cloudlog.go

```diff
@@ -19,60 +19,39 @@ import (
 	"fmt"
 	"log/slog"
 	"os"
-	"os/signal"
 	"runtime"
-	"syscall"
-	"time"
 
 	"cloud.google.com/go/logging"
 	"cloud.google.com/go/logging/apiv2/loggingpb"
 	"go.opentelemetry.io/otel/trace"
 )
 
 // SetGlobal sets the global slog to use output a GCP cloud logging and stderr
-func SetGlobalLogger(ctx context.Context, project, logName string) error {
+func SetGlobalLogger(ctx context.Context, project, logName string, l slog.Level) func(context.Context) error {
 	cl, err := logging.NewClient(ctx, fmt.Sprintf("projects/%s", project))
 	if err != nil {
-		return err
+		return func(ctx context.Context) error { return nil }
```

Comment:
Now we are returning a func that returns no error, Is this WAI?

Reply:
i want this to be non-fatal if it fails, so it becomes a noop on error